lld/COFF/Chunks.h

```diff
@@ -749,6 +749,17 @@ class ECCodeMapChunk : public NonSectionChunk {
   std::vector<ECCodeMapEntry> &map;
 };
 
+class CHPERedirectionChunk : public NonSectionChunk {
```

Comment:
Is this documented somewhere as being named with `CHPE`, as opposed to ARM64EC in some form?

Reply:
Thanks, if this is referred to in the load configs with a CHPE prefix, using these names is totally fine with me.

(I guess historically, the naming may stem from which parts were used in CHPEv1 and which are new for ARM64EC etc.)